Core/Main/PTMagic.cs

```diff
@@ -1518,15 +1518,15 @@ private void ApplySingleMarketSettings()
 
             // Check ignore markets
             List<string> ignoredMarkets = SystemHelper.ConvertTokenStringToList(marketSetting.IgnoredMarkets, ",");
-            if (ignoredMarkets.Contains(marketPair))
+            if (ignoredMarkets.Any(marketPair.Contains))
```

Comment:
I think it would be best to only match the beginning of the pair string to avoid partial matches within the string, and to be case insensitive. This is my suggested match critera:

if (ignoredMarkets.Any(im => marketPair.StartsWith(im, StringComparison.InvariantCultureIgnoreCase)))